### PR TITLE
crimson/common: instantiate interrupt_cond in .cc

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
jenkins test make check arm64

File: cmake/modules/BuildFIO.cmake [hunk 1]
```diff
@@ -16,7 +16,7 @@ function(build_fio)
     GIT_REPOSITORY "https://github.com/axboe/fio.git"
     GIT_CONFIG advice.detachedHead=false
     GIT_SHALLOW 1
-    GIT_TAG "fio-3.15"
+    GIT_TAG "fio-3.27"
```
Comment (tchaikov) [inline cmake/modules/BuildFIO.cmake:19]: @ifed01 hi Igor, could you help review this change?

**Comment by tchaikov** [comment]
due to the name conflict, we cannot bump the fio version yet.

**Review by rzarzynski — APPROVED** [review]

**Pull request merged**